civihr_default_theme/assets/sass/partials/components/_scrollbar.scss

```diff
@@ -0,0 +1,17 @@
+// Custom scrollbar
+// --------------------------------------------------
+.chr_custom-scrollbar {
+    &::-webkit-scrollbar {
+        height: 10px;
+    }
+
+    &::-webkit-scrollbar-track {
+        background-color: #F5F5F7;
```

Comment:
_(Applies for every previous/next instance)_

Is any of these values already present as a sass variable? if not, is it worth creating a variable for them?